smack-websocket/src/main/java/org/jivesoftware/smack/websocket/WebsocketException.java

```diff
@@ -0,0 +1,28 @@
+/**
+ *
+ * Copyright 2020 Aditya Borikar
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jivesoftware.smack.websocket;
+
+import java.util.Set;
+
+public class WebsocketException extends Exception {
+    private static final long serialVersionUID = 1L;
+    Set<Throwable> throwableSet;
```

Comment:
I know that it is tempting to use a `Set` here, but IMHO the right container is `List` for this case.